bsp_diff/caas/vendor/intel/external/onevpl-intel-gpu/0002-onevpl-driver-changes-for-rpl-s-platform.patch

```diff
@@ -1,141 +0,0 @@
-From 3705a4970e27281f7f379ed8a0509dc5223b5ddb Mon Sep 17 00:00:00 2001
```

Comment:
This patch is nullifed by 0003-Revert-onevpl-driver-changes-to-support-rpl-s-platfo.patch
so removed both of them